src/util-ebpf.c

```diff
@@ -645,10 +645,15 @@ static int EBPFForEachFlowV6Table(LiveDevice *dev, const char *name,
         return 0;
     }
 
+    uint64_t pkts_cnt = 0;
```

Comment:
the role of pkts_cnt is very unclear here.

Reply:
the whole function logic is quite confusing to me. Can you explain it more?

Reply:
Fixed by updating return of the EBPFOpFlowForKey function.